python/CMakeLists.txt

```diff
@@ -46,5 +46,5 @@ endif()
 
 add_custom_target(python_wheel DEPENDS ngraph ${CMAKE_CURRENT_BINARY_DIR}/dist/)
 if (NGRAPH_CPU_ENABLE)
-    add_dependencies(python_wheel libdnnl)
```

Comment:
You made the alias libdnnl but then never use it. Either remove alias or use it.

Reply:
libdnnl is used inside external_onednn.cmake. DNNL::dnnl is the alias created out of it. I'm using the name DNNL::dnnl to match the target oneDNN exposes. That would make using prebuilt oneDNN easier in the future.